pkg/v1/providers/config_default.yaml

```diff
@@ -174,9 +174,9 @@ NSXT_PASSWORD: ""
 #! NSX-T host
 NSXT_MANAGER_HOST: ""
 #! set this to true if NSX-T uses self-signed cert
-NSXT_ALLOW_UNVERIFIED_SSL: "false"
+NSXT_ALLOW_UNVERIFIED_SSL: false
```

Comment:
@tenczar : this change enables legacy config file not specifying this value to work correctly with the vsphere CC, but  this changes the behavior when we need to use the equivalent legacy config file in fallback (pure client side ytt) mode. 

Either existing logic that assumed string instead of bool will have to be updated (see ytt/02_addons/cpi/cpi_addon_data.lib.yaml and associated vendir-ed code)

or we should consider retaining the definition/semantics of the variables in config_default.yaml

(same for NSXT_REMOTE_AUTH)

Reply:
>Either existing logic that assumed string instead of bool will have to be updated (see ytt/02_addons/cpi/cpi_addon_data.lib.yaml and associated vendir-ed code)
If we change the existing logic, will the upgrade break, since customer's previous value is still string

Reply:
Thanks for reminding me. We will look for convertion for CC to avoid breaking existing logic.